### DISCUSSION TITLE
Just to report: Invariant test fail, then I "forge clean", then it pass!?

### DISCUSSION THREAD

**Discussion by CH-coding-fire** [Cyfrin/foundry-full-course-cu#1962] [General]
Hi all,
As per the title, this is what happened. I don't know why, just to report. "forge clean" is for cleaning cache according to official document,

**Comment by EngrPips** [comment]
Yeah, `forge clean` clears all cached data and runs all processes as fresh processes. That seems to solve problems at times. Thanks for the head-ups

**Reply by AllenGeorge08** [reply]
Yo , Thanks @CH-coding-fire ..Worked for me too. 

Saved me hours.

**Reply by EngrPips** [reply]
Awesome, Keep crushing it, buddy.
